IPython/terminal/prompts.py

```diff
@@ -6,6 +6,13 @@
 
 from IPython.core.displayhook import DisplayHook
 
+try:
+    from prompt_toolkit.token import ZeroWidthEscape
+except ImportError:
+    # backward compat with PTK 1.x that does not yet define this:
```

Comment:
Is this only going into 2.x? Our requirement is currently for `prompt_toolkit>=1.0.0,<2.0.0`, so this code shouldn't be run with PTK 2. Or is the comment erroneous, and ZWE will be available in a future PTK 1.x release?